css/styles.css

```diff
@@ -0,0 +1,264 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.wrapper {
+  height: 5410px;
+  width: 375px;
+  left: 11282px;
+  top: -1883px;
+  background: #f7f7f9;
+}
```

Comment:
- [ ] The Figma template background color of this section of the application is `#fff`. Please use the correct background color to match the template.